core/client

```diff
@@ -1 +1 @@
-Subproject commit 61fe83dff375a145d5272a5382aa2c72bd994830
+Subproject commit 4b3bc50b04cd28f303e6981e2f027c3eef24e592
```

Comment:
Small question in general:
I think in general we should try to avoid updating `core/client` or?